src/content/blog/whats-new-march-2024.mdx

```diff
@@ -0,0 +1,193 @@
+- We launched the all new [Astro Developer Portal](/blog/dev-portal)! 🎉
+- All tests in Astro have been [migrated to `node:test`](/blog/node-test-migration/)
+
+## Releases
+
+- Astro [4.5](https://astro.build/blog/astro-450/)
+- The first release of [`astro:db`](blog/astro-db-deep-dive/)
+- Starlight [0.21](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.21.0)
+
+## Docs
+
+This month, we added a new [Upgrade Astro](https://docs.astro.build/en/upgrade-astro/) page to the docs that includes how to upgrade to the latest version, past upgrade guides and docs for older versions, as well as an explanation of Astro's semantic versioning from our platform team. If you ever wondered what all the different numbers in our release versions meant, we've now
+got you covered!
+
+Our weekly Discord call “Talking and Doc’ing” is a chance for you to watch Team Docs work in public, jump in a shared Codespace or Gitpod Workspace, and contribute to the docs live, together. This month’s topics included:
+
+- Updating our [community website](https://community.astro.build) with all of our past community award winners going back to the very beginning, giving us a chance to test a new pagination component.
+- Adding our new `<Steps />` component to the entire Build a Blog beginner tutorial.
+- Helping a brand new Astro-community tips and resource site figure out how to get started with Starlight. Check out [astro-tips.dev](https://astro-tips.dev) for recipes and more!
+
+## Community
+
+Astro wouldn’t be what it is without its community. From Discord support, to documentation improvements, to translations, to code contributions, Astro welcomes it all.
+
+Hear some of our past community award winners share what it's like contributing to Astro, and see for yourself what makes the Astro experience so special. 
+
+<YouTube id='https://www.youtube.com/watch?v=TtRtkTzHVBU' />
+
+### Community Awards February
+
+These are some of the people who went above and beyond last month to make all our lives just a little (or a lot!) better:
+
+- 🌟 $500 to @mingjunlu for contributing to all aspects of the Astro project and community including support, translations, migrating tests and fixing issues.
+- 🍾 $250 to @ktym4a for contributions to the Astro ecosystem, including core and integrations.
+- 🎈 $250 to @jackshelton Shelton for outstanding leadership and community support with the QwikDev/astro integration.
+- 🎊 $250 to @soslan117 for their enthusiastic contributions translating Astro Docs into Russian.
+- 🎉 $250 to @bryceguy for creating several popular community packages and leadership in community support.
+
+👕 ✨ Swag shop merch issued this month to @tigger_hyk, @nathanlawsn, and @sergios1630 !
+
+Check them out and learn more about the Astro Community Awards on [our community website](https://community.astro.build).
+
+## Showcase
+
+### Content
+
+Because you can never get tired of watching and reading Astro Content.
```

Comment:
```suggestion
Featured: "The Story of Astro DB"

<YouTube id=1https://www.youtube.com/watch?v=aUfYzjKNCh4' />

Because you can never get tired of watching and reading Astro Content.
```